src/action.js

```diff
@@ -127,7 +128,29 @@ const action = async () => {
     );
   }
   console.log("✅ CodeScanningAlerts data sent to Azure Log Analytics");
+
+  // Get Renovate PRs data for current repo
+  const renovatePRsData = await queryRenovatePRs(octokit, owner, repo);
+
+  // Breaks code scanning results into chunks of 10
+  const renovatePRsDataChunks = renovatePRsData.renovate_prs;
+
+  for (let i = 0; i < renovatePRsDataChunks.length; i += chunkSize) {
```

Comment:
Since `const chunkSize = 10;` is now used in two places, could you hoist it up to just under the `const prefix` value on line 18.

Reply:
done